pkg/kubelet/kubelet.go

```diff
@@ -281,8 +282,12 @@ func NewMainKubelet(
 		pods:                           pods,
 		syncLoopMonitor:                util.AtomicValue{},
 		resolverConfig:                 resolverConfig,
+		nodeLabels:                     NewNodeLabelManager(hostname, nodeLabelPluginDir),
 	}
 
+	// Start polling for node labels
+	klet.nodeLabels.Run()
```

Comment:
Please comment that the first iteration of node label updates is synchronous. It's written in the node label manager, but here it's important and not obvious from the function name.